R/classicProcess.R

```diff
@@ -1926,6 +1928,54 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
   }
 }
 
+
+.procRsquared <- function(options, modelsContainer) {
```

Comment:
```suggestion
.procRsquared <- function(jaspResults, options, modelsContainer) {
```

It is convention to store all results in `jaspResults`. The object `modelsContainer` is also stored in `jaspResults` which is why it also works like this, however, I think it would be better to directly store the table in `jaspResults` because `modelsContainer` is supposed to contain only the objects stored for each individual model and no output.